docs/book/src/cronjob-tutorial/pprof-tutorial.md

```diff
@@ -0,0 +1,61 @@
+# Using Pprof for Controller Profiling
+
+Pprof is a Go based library designed for profiling the binary in order to gather CPU, and Memory related data. Pprof is helpful in finding any sort of bottlenecks that are appearing either at Memory, CPU, or both levels.
+
+Pprof itself runs as part of the the Controller. You can extract the statistics, and visualize them on an interactive website. Furthermore, **you don't need to install this package seperately**. Pprof comes packaged as part of Kubebuilder `controller-runtime` [dependency](https://github.com/kubernetes-sigs/controller-runtime/pull/1943) within the scafolded project.
+
+For further information about the Pprof usage, check out the official [docs](https://github.com/google/pprof).
+
+# How to use Pprof in your Codebase
+
+1. In your `cmd/main.go` file, specify the `PprofBindAddress` URL field as part of Controller Manager's configurations:
+
+    ```golang
+    mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+      ....
+      Scheme:                 scheme,
+      LeaderElection:         enableLeaderElection,
+      // PprofBindAddress is the TCP address that the controller should bind to
```

Comment:
```suggestion
      // PprofBindAddress is the TCP address that the controller should bind to
```

We do not need those options right?

Reply:
Nope. Only `PprofBindAddress` is required.